add ttl

bitner opened the pull request:
Adds a TTL parameter to be able to control Cache Control headers.

vincentsarago commented:
hey sorry @bitner I didn't see the notification for this 😬 

could you add some documentation somewhere on the docstrings or on the readme (I know that I need to do the same :D )

bitner commented:
No problem. This is something I added in my fork that seemed more broadly useful. 

I added a section in the readme already. Changes to proxy.py are kind of all over the place along with all the other parameters, not sure inline comments wouldn't just get in the way. Happy to add more or change up the docs that are there in the readme if you'd like things differently.

vincentsarago commented:
@bitner 🤦‍♂️ my 👀 are not good at this time. Thanks for the PR 👼
